### PR TITLE
Add travel bans from 2020-04-03

### PR DESCRIPTION
**Pull request by mertd**
I also moved the travel band json into its own file as it's gotten quite long

Related: #84 

@reustle

### CODE REVIEW

**Comment by reustle** [comment]
You're a star! Thanks so much. Please mark it as 'ready for review' when you'd like it checked. Cheers :)

**Comment by mertd** [comment]
@reustle should be ready :)

File: src/index.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ import locI18next from "loc-i18next";
 import translationEn from "./i18n/en.json";
 import translationJa from "./i18n/ja.json";
 
+// import static data
+import travelRestrictions from "./data/travelRestrictions"; // refer to the keys under "countries" in the i18n files for names
```
Comment (ann-kilzer) [inline src/index.js:21]: Nice 👍

**Comment by reustle** [comment]
Thanks! Do you know if there is a page we can link to instead of a PDF? Also wondering if it makes sense in the near future to link tot he local embassy website for each country (not sure if they're even updating this info yet for some).

**Comment by mertd** [comment]
I guess the PDF is the most official source.

japan.travel also maintains a list though: https://www.japan.travel/en/coronavirus/

**Pull request merged**